docs(generic): update importers docstrings, comments

koeaw opened the pull request:
Fix grammar, spelling, punctuation; add paragraphs to docstrings, make first line a (useful) summary
line following PEP 257 recommendation.

b1rger commented on apis_core/generic/importers.py:
```diff
@@ -17,14 +17,14 @@
-    By default it fetches a resource, first tries to parse it using
-    our rdf parser, if that fails tries to parse it using json and
+    A generic importer class which provides standard methods for
+    importing data from a URI and creating a model instance off it.
```
```suggestion
    importing data from a URI and creating a model instance of it.
```

koeaw replied:
"of" would be applicable if a model instance were a different _state_ or a true representation of the data. That's not the case though, is it? What we're doing is creating model instances _based on_ (some of) the data.

koeaw replied:
"off of" or "from" are alternatives which could be substituted

koeaw replied:
> > I prefer "from"
> 
> Ok, will change it to that.

But I'm closing this as resolved/invalid now already, because "of" isn't grammatically correct either way and "from" instead of "off" is a preference.

b1rger commented on apis_core/generic/importers.py:
```diff
@@ -17,14 +17,14 @@
-    an URI and creating a model instance of it.
-    By default it fetches a resource, first tries to parse it using
-    our rdf parser, if that fails tries to parse it using json and
+    A generic importer class which provides standard methods for
```
```suggestion
    A generic importer class which provides the standard methods for
```

koeaw replied:
These methods aren't an existing, universally known/accepted or expected thing, though, but something we've thought up/come up with ourselves, right? The definite article only makes sense in reference to known quantities.

Same for "standard" actually.

koeaw replied:
I'm closing this as resolved based on your thumbs-up, @gythaogg..

koeaw commented:
@gythaogg I changed "off" to "from" and also removed the word "standard" as per https://github.com/acdh-oeaw/apis-core-rdf/pull/1789#discussion_r2041933451.
